File: incidents-parking/old/api-parkinglots/src/domain/models/parking.ts
export interface Parking {
  id: string; // UUID in PostgreSQL
  country: string;
  name: string;
  latitude: number;
  longitude: number;
  address: string;
  availableSpace?: number;
  idStatus: string;
  createdAt: number;
  updatedAt: number;
}

File: incidents-parking/services/crashsaver-api-parking/types/parking.go
package types

import (
	"crashsaver/parking/data"

	"github.com/google/uuid"
)

type CreateParking struct {
	Country        string                  `json:"country"`
	Name           string                  `json:"name"`
	Address        string                  `json:"address"`
	Language       string                  `json:"language"`
	Latitude       float64                 `json:"latitude"`
	Longitude      float64                 `json:"longitude"`
	AvailableSpace int32                   `json:"availableSpace"`
	Emails         []string                `json:"emails"`
	ServicesIds    []uuid.UUID             `json:"servicesIds"`
	Hours          data.PkPriceXHoursArray `json:"hours"` //{hours: int, price: float64}
}
